modules/troubleshooting-enabling-kdump-day-one.adoc

```diff
@@ -66,6 +66,7 @@ systemd:
 <2> Provide kernel arguments to reserve memory for the crash kernel. You can add other kernel arguments if necessary.
```

Comment:
wondering if we can make the `crashkernel` suggestion here as well for power

Reply:
Agree. Added the crashkernel suggestion for IBM Power.

fyi: @mkumatag